config/locales/en.yml

```diff
@@ -40,3 +40,11 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    coauthor_notification:
+      subject: "[%{app_name}] Co-Author Notification"
+      edit_the: "Edit the %{type}"
```

Comment:
We need to make it clear to translators what type can be - how about "edit_the_work_chapter_or_series"? It's long, and there is virtue in brevity, but they'll need the clarity.